app/v3/utils/getRandomTestData.ts

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License.
+
+import * as crypto from 'crypto';
+import { delay, delaySync } from './delay';
+
+export function getRandomTestData(): string {
+    // This should start with non-numeric data to prevent this bug from causing a test failure
+    // https://github.com/Azure/azure-functions-nodejs-library/issues/90
+    return `testData${getRandomHexString()}`;
+}
+
+export function getRandomHexString(length = 10): string {
+    const buffer: Buffer = crypto.randomBytes(Math.ceil(length / 2));
+    return buffer.toString('hex').slice(0, length);
+}
+
+export function getRandomInt(min: number, max: number): number {
+    return Math.floor(Math.random() * (max - min) + min);
+}
+
+export function getRandomBoolean(percentTrue: number): boolean {
+    return Math.random() * 100 > percentTrue;
+}
+
+export async function addRandomDelay(): Promise<void> {
+    if (getRandomBoolean(95)) {
```

Comment:
I have a couple questions here. First, why set the parameter to getRandomBoolean() to 95? Second, why add the if/else blocks and not just go straight to delay(getRandomInt(0, 250))?

Reply:
I'm trying to simulate somewhat realistic scenarios and I wanted a combination of asynchronous and synchronous delays. 95 means return true 95% of the time, which means the vast majority of the time it will be an async delay. I hope most of our users use async, plus if I do sync too often the test will timeout (running tests in parallel with async is inherently much faster)

Reply:
Ah, so we're saying it's not just random in terms of the length of time, but also in the nature of the delay. Would you be open to a name change suggestion? Maybe, `addRandomAsyncOrSyncDelay`? That's very C#-ish, I know.

Reply:
fixed